### DISCUSSION TITLE
Can I use NOPMD for a line in visualforce?

### DISCUSSION THREAD

**Discussion by babula** [pmd/pmd#3267] [Q&A]
I'm trying to suppress the VfUnescapeEl on a specific line in a visualforce page and having trouble figuring out the best way to handle it. I assumed adding `<!-- NOPMD -->` to the line would do the trick but sadly I was wrong. I looked through the documentation but didn't find anything. Any recommendations?

**Comment by adangel** [comment]
I'm afraid that's not supported (yet). The parser would need to be updated to support this (e.g. exposing eventually the NOPMD comments here: https://github.com/pmd/pmd/blob/dca0bf9545f95010fdfe534d54cac09f31c4118f/pmd-visualforce/src/main/java/net/sourceforge/pmd/lang/vf/VfParser.java#L58)

I assume, you are talking about html like comments (`<!-- NOPMD ... -->`), is this correct?

**Reply by babula** [reply]
Oops, just noticed that my code sample got commented out of my initial question. Yes though, that is exactly what I was thinking.
